pgml-dashboard/src/api/cms.rs

```diff
@@ -550,7 +550,7 @@ This is the end of the markdown
         let rsp = req.dispatch().await;
 
         assert!(
-            rsp.status() == Status::NotFound,
+            rsp.status() == Status::Ok,
```

Comment:
Should be 404 when doc not found. If we return ok, google will index the not found page and real and pollute the search index.

Reply:
Ok, I can switch it back.  We will have to wait on keeping the lefthand nav on bad path then.

Reply:
I don't understand what the connection between this and the leftnav is with the current context.

Reply:
After a bad path Montana would like to other docs there on the left to easy navigate away from the 404 response. 


![Screenshot 2023-12-20 at 4 00 03 PM](https://github.com/postgresml/postgresml/assets/39170265/330e66a0-951f-45d3-aa62-1ab2cd2f3272)

Reply:
Ah I see, you'll just need a custom 404 page for the docs instead of using the default one. You can set that up with a 404 catcher in rocket.